src/interfaces/store-interfaces/state-interface.ts

```diff
@@ -1,14 +1,13 @@
-import { CertifyIF } from '../stepper-interfaces/ReviewConfirm/certify-interface'
-import { DefineCompanyIF } from '../stepper-interfaces/DefineCompany/define-company-interface'
+import { CertifyIF, DefineCompanyIF, TombStoneIF } from '@/interfaces'
 
 // State model example
 export interface StateModelIF {
-  stateText: string | undefined
-  currentDate: string | null
-  certifyState: CertifyIF
-  currentStep: number
-  isSaving: boolean
-  isSavingResuming: boolean
-  isFilingPaying: boolean
-  defineCompanyStep: DefineCompanyIF
+  tombstone: TombStoneIF,
```

Comment:
I thought you would merge the tombstone properties with the state properties, but I guess this is OK too.

Reply:
ohh i see, i can put them at root level of the state as well. I wasn't sure if it was still as a subset of sorts.